src/main/java/com/picolite/controllers/ArticleController.java

```diff
@@ -0,0 +1,49 @@
+package com.picolite.controllers;
+
+
+import com.picolite.fakedb.FakeDB;
+import com.picolite.models.Article;
+import com.picolite.models.ArticleContainer;
+import com.picolite.tools.FancyText;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class ArticleController {
+
+
+    @CrossOrigin
+    @RequestMapping(value = "/articles/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<Article> getArticle(@PathVariable("id") Long articleId)
+    {
+        System.out.println("Retrieving article " + articleId);
+
+        Article a = FakeDB.fetchArticle(articleId);
+
+        //punch up content
+        a.setContent(FancyText.convert(a.getContent()));
+
+        return new ResponseEntity<Article>(a, HttpStatus.OK);
+    }
+
+    @CrossOrigin
+    @RequestMapping(value = "/articles", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Would be good to add specification on your RequestMappings with the type of call (GET, POST, DELETE, PUT, etc) you're making, example this would be @RequestMapping(method = RequestMethod.GET)